### PR TITLE
Task/add chain id 2 addr map

### PR DESCRIPTION
**Pull request by 007vasy**
add chain ids to other libs like:
- https://github.com/banteg/multicall.py/issues/49
- https://github.com/DefiLlama/defillama-sdk/issues/80

have easier time adding new chains

### CODE REVIEW

**Comment by 007vasy** [comment]
@mds1 added chain ids (so I can add the current and future contracts more easily to other chains)

File: README.md [hunk 1]
```diff
@@ -39,58 +39,58 @@ Multicall3 contains the following improvements over prior multicall contracts:
...
+| Cronos                  | 25                                 | [0xcA11bde05977b3631167028862bE2a173976CA11](https://cronoscan.com/address/0xca11bde05977b3631167028862be2a173976ca11#code)                                      |
+| Fuse                    | 122                                | [0xcA11bde05977b3631167028862bE2a173976CA11](https://explorer.fuse.io/address/0xcA11bde05977b3631167028862bE2a173976CA11/contracts)                              |
+| Songbird Canary Network | 19                                 | [0xcA11bde05977b3631167028862bE2a173976CA11](https://songbird-explorer.flare.network/address/0xcA11bde05977b3631167028862bE2a173976CA11/contracts)               |
+| Coston Testnet          | 16                                 | [0xcA11bde05977b3631167028862bE2a173976CA11](https://coston-explorer.flare.network/address/0xcA11bde05977b3631167028862bE2a173976CA11/contracts)                 |
+| Coston2 Testnet         | -                                  | [0xcA11bde05977b3631167028862bE2a173976CA11](https://coston2-explorer.flare.network/address/0xcA11bde05977b3631167028862bE2a173976CA11/contracts)                |
```
Comment (mds1) [inline README.md:73]: Using the RPC of https://coston2-api.flare.network/ext/C/rpc, looks like this is 114

File: README.md [hunk 2]
```diff
@@ -39,58 +39,58 @@ Multicall3 contains the following improvements over prior multicall contracts:
...
+| Boba                    | 288                                | [0xcA11bde05977b3631167028862bE2a173976CA11](https://blockexplorer.boba.network/address/0xcA11bde05977b3631167028862bE2a173976CA11/contracts)                    |
+| Aurora                  | 1313161554                         | [0xcA11bde05977b3631167028862bE2a173976CA11](https://explorer.mainnet.aurora.dev/address/0xcA11bde05977b3631167028862bE2a173976CA11)                             |
+| Astar                   | 592                                | [0xcA11bde05977b3631167028862bE2a173976CA11](https://blockscout.com/astar/address/0xcA11bde05977b3631167028862bE2a173976CA11/contracts)                          |
+| OKC                     | -                                  | [0xcA11bde05977b3631167028862bE2a173976CA11](https://www.oklink.com/en/okc/address/0xca11bde05977b3631167028862be2a173976ca11)                                   |
```
Comment (mds1) [inline README.md:77]: RPC of https://exchainrpc.okex.org gives chain ID of 66

**Review by mds1 — COMMENTED** [review]
Been planning to add this info, so thanks a lot for doing it!

**Comment by 007vasy** [comment]
added them

found OKC by chain id, but Coston2 is not on the list yet, hence the ()

**Comment by 007vasy** [comment]
@mds1 reordered the chains by id, this way it's way easier to add new ones

**Review by mds1 — APPROVED** [review]
Thanks!

**Pull request merged**